moeflow_worker.py

```
@@ -0,0 +1,197 @@
+                'pts': o.lines,
+                'text': o.text,
+                'textlines': self.default(o.texts),
+            }
+        if isinstance(o, utils_generic.Quadrilateral):
+            return {
+                'pts': o.pts,
+                'text': o.text,
+                'prob': o.prob,
+                'textlines': self.default(o.textlines),
+            }
+        elif isinstance(o, filter) or isinstance(o, tuple):
```

Ruff

Multiple `isinstance` calls for `o`, merge into a single call

Merge `isinstance` calls for `o`

(SIM101)